jbpm-wb-human-tasks/jbpm-wb-human-tasks-client/src/test/java/org/jbpm/workbench/ht/client/editors/taskslist/TaskAdminViewTest.java

```diff
@@ -0,0 +1,257 @@
+/*
+ * Copyright 2017 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.jbpm.workbench.ht.client.editors.taskslist;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+
+import com.google.gwt.view.client.AsyncDataProvider;
+import com.google.gwtmockito.GwtMockitoTestRunner;
+import org.dashbuilder.dataset.DataSetOp;
+import org.dashbuilder.dataset.DataSetOpType;
+import org.dashbuilder.dataset.filter.ColumnFilter;
+import org.dashbuilder.dataset.filter.DataSetFilter;
+import org.gwtbootstrap3.client.ui.Button;
+import org.jboss.errai.security.shared.api.identity.User;
+import org.jbpm.workbench.df.client.filter.FilterSettings;
+import org.jbpm.workbench.df.client.list.base.DataSetEditorManager;
+import org.jbpm.workbench.df.client.list.base.DataSetQueryHelper;
+import org.jbpm.workbench.common.client.experimental.grid.base.ExtendedPagedTable;
+import org.jbpm.workbench.ht.client.editors.taskslist.TaskAdminListPresenter;
+import org.jbpm.workbench.ht.client.editors.taskslist.TaskAdminListViewImpl;
+import org.jbpm.workbench.ht.client.editors.taskslist.TaskListViewImpl;
+import org.jbpm.workbench.ht.model.TaskSummary;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.ArgumentCaptor;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.invocation.InvocationOnMock;
+import org.mockito.stubbing.Answer;
+import org.uberfire.ext.services.shared.preferences.GridColumnPreference;
+import org.uberfire.ext.services.shared.preferences.GridGlobalPreferences;
+import org.uberfire.ext.services.shared.preferences.GridPreferencesStore;
+import org.uberfire.ext.services.shared.preferences.MultiGridPreferencesStore;
+import org.uberfire.ext.services.shared.preferences.UserPreferencesService;
+import org.uberfire.ext.widgets.common.client.tables.FilterPagedTable;
+import org.uberfire.ext.widgets.table.client.ColumnMeta;
+import org.uberfire.mocks.CallerMock;
+import org.uberfire.mvp.Command;
+
+import static org.junit.Assert.*;
+import static org.mockito.Mockito.*;
+import static org.jbpm.workbench.ht.model.TaskDataSetConstants.*;
+
+@RunWith(GwtMockitoTestRunner.class)
+public class TaskAdminViewTest {
```

Comment:
TaskAdminViewTest and TasksListViewTest seem to share a fair bit of logic, perhaps you could also create a `AbstractTaskListViewTest` class to share the common code.